paywall/src/unlock.js/startup.ts

```diff
@@ -30,16 +30,19 @@ export default function startup(window: UnlockWindow) {
 
   const dataIframe = makeIframe(
     window,
-    process.env.PAYWALL_URL + '/static/data-iframe.1.0.html' + origin
+    process.env.PAYWALL_URL + '/static/data-iframe.1.0.html' + origin,
+    'unlock data'
   )
   const checkoutIframe = makeIframe(
     window,
-    process.env.PAYWALL_URL + '/checkout' + origin
+    process.env.PAYWALL_URL + '/checkout' + origin,
+    'unlock checkout'
```

Comment:
Potentially dumb Q, but should there be some uniqueness to these launched names? eg if multiple iframes are eventually used?

Reply:
at this point, the 3 names used are `unlock data`, `unlock checkout`, and `unlock accounts`. I wonder if the confusion is the space in the name makes it look like all 3 are `unlock`?